src/views/CardViewPage.vue

```diff
@@ -76,7 +71,7 @@ export default {
       OverpoweredVotes: 0,
       FairEnoughVotes: 0,
       Nerflevel: 0,
-      Owner: "",
+      Owner: " ",
```

Comment:
why are we doing this? is the aim that !owner should not evaluate true? or is  it so that the route param has a non empty string?

Reply:
The second, if the route parameter is initially empty, the router-link won't work at all.